configuration/processors.md

```diff
@@ -115,15 +116,16 @@ Dimension counter supports exact same configuration as **Simple Keyword Match**
 | Key | Description | Required |
 | :--- | :--- | :--- |
 | name | User defined name of this specific processor, used for mapping this processor to a workflow. | Yes |
-| pattern | Regular Expression pattern containing a named capture group representing the dimension. _Note: named capture groups must follow Golang regex protocol, ex:  "status\_code=\(?P&lt;status\_code&gt;\d+\)"_ | Yes |
+| pattern | Regular expression pattern containing a named capture group representing the dimension. _Note: named capture groups must follow Golang regex protocol, ex:  "status\_code=\(?P&lt;status\_code&gt;\d+\)"_ | Yes |
```

Comment:
/s/_Note/Note

Reply:
_Note makes it italic but that's not the only way. Surrounding with single stars '\*Note\*' also makes it italic. I'm using stars in other places so also changing this